docs/components/Footer.js

```diff
@@ -4,40 +4,80 @@ import { Box, Flex, Icon, Link, Text } from 'gestalt';
 import trackButtonClick from './buttons/trackButtonClick.js';
 
 export default function Footer(): Node {
+  const links = [
+    { title: "What's new", url: 'https://gestalt.pinterest.systems/home' },
+    { title: 'About us', url: 'https://gestalt.pinterest.systems/about_us' },
+    { title: 'Careers', url: 'https://www.pinterestcareers.com/', external: true },
```

Comment:
Consider pre-filling the search: https://www.pinterestcareers.com/job-search-results/?keyword=gestalt